packages/liveblocks-react-ui/.eslintrc.js

```diff
@@ -18,7 +18,28 @@ module.exports = {
     // -------------------------------
     // Custom syntax we want to forbid
     // -------------------------------
-    "no-restricted-syntax": ["error", ...commonRestrictedSyntax],
+    "no-restricted-syntax": [
+      "error",
+      ...commonRestrictedSyntax,
+      {
+        selector:
+          "ImportDeclaration[source.value='react'] ImportSpecifier[imported.name='useSyncExternalStore']",
+        message:
+          "useSyncExternalStore is only available on React >=18. Import it from 'use-sync-external-store/shim/index.js' instead.",
+      },
+      {
+        selector:
+          "ImportDeclaration[source.value='react'] ImportSpecifier[imported.name='useId']",
+        message:
+          "useId is only available on React >=18. Import it from '/src/utils/use-id' instead.",
+      },
+      {
+        selector:
+          "ImportDeclaration[source.value='react'] ImportSpecifier[imported.name='useTransition']",
+        message:
+          "useTransition is only available on React >=18. Import it from '/src/utils/use-transition' instead.",
+      },
```

Comment:
Shall we add `use` here too, while at it?

Reply:
Since we don't have a polyfill usable in `react-ui` for it yet I wouldn't know what to recommend to fix the error?

Reply:
Updated in https://github.com/liveblocks/liveblocks/pull/1774/commits/e99a0e3e2e5738c8bb570d491282c436582233d7.